elastic/integrations
# [syslog_router] Add syslog router integration

## Proposed commit message

- Add initial implementation of the syslog router integration which will identify and route syslog-based events to security integrations.

## Checklist


## How to test this PR locally

```
cd packages/syslog_router
elastic-package test
```

> [!WARNING]
> Due to limitations in `elastic-package`, automated tests cannot verify routing behavior to data streams external to this package. Verification will have to be performed manually. To verify, follow the instructions in the integration on installing assets for another integration (Cisco ASA, for example), configure the integration, and send a relevant log to the Agent.

## Related issues

- Closes (TODO: integrations issue)
- Relates elastic/security-team#10081
